lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -12,6 +12,9 @@ var combineData = require('../Core/combineData');
 var loadView = require('../Core/loadView');
 var removeView = require('../Core/removeView');
 var ChartViewModel = require('./ChartViewModel');
+var cContainer = document.getElementById('cesiumContainer');
```

Comment:
Hi, thanks for this Chloe - looks good. Yes, is there a way to get the cesium container direct from the terria object?